src/server/app.js

```diff
@@ -27,4 +29,6 @@ app.get('/organizations/:organizationId/events/:eventId/rides/:id', routes.getRi
 
 app.put('/api/organizations/:organizationId/events/:eventId/rides/:rideId', routes.updateRidePassengers);
 
+app.use('/graphql',  graphqlHTTP({schema: schema, graphiql: true}));
```

Comment:
This provides REALLLY REALLLY REALLY easy to use tool for graphql app (typeahead and stuff... you really want it)